scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -87,7 +87,7 @@ object ScalafmtRunner {
     val scala213 = Scala213
       .withAllowTrailingCommas(true)
     val default = scala213
-    val Scala3 = Dotty
+    val Scala3 = Dotty.withAllowSignificantIndentation(false)
 
```

Comment:
this actually duplicates the change in extension group. probably should have been submitted as a separate pr.

Reply:
@tgodzik we might merge #2298 with the above change first or you can extract it into separate pr as suggested by @kitbellew

Reply:
We can just rebase one of the PRs, should be easy enough.